easybuild/easyconfigs/f/fineRADstructure/fineRADstructure-20210514-foss-2023a.eb

```
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'fineRADstructure'
+version = '20210514'
+
+homepage = 'http://cichlid.gurdon.cam.ac.uk/fineRADstructure.html'
+description = "A package for population structure inference from RAD-seq data"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

doesn't look like the software uses blas/fftw or MPI, so this could be dropped down to `GCC` level
```suggestion
toolchain = {'name': 'GCC', 'version': '12.3.0'}
```

---

note that the filename would need updating to match as well